docs/getting-started/products/yvaults/v3.md

```diff
@@ -0,0 +1,245 @@
+# Vaults Version 3
+
+Version 3 yVaults iterates on Version 2 by increasing robustness and developing yearn’s path towards further decentralization, while keeping the same proven product (yield-bearing tokens) that abstract builders and users from the hard work of yield farming. Version 3 will initially have the same functionality as Version 2, but the vaults are more robust and modular.
+
+Version 3 yVaults improves on Version 2 by:
+- Increasing vault modularity, allowing for smaller and safer pieces of code.
+- Simplify strategy creation, empowering strategists and reducing the chance for errors.
+- Adding new products like ySwaps which increases swap efficiency and yJuniorTranches which allows for different risk profiles.
+
+## Smart Modules
+
+Smart Modules implement logic to be iterated until it can be made immutable. If any Smart Module fails, the vault can live without them just enough to return funds to depositors. **When Version 3 launches, the Smart Modules will replicate Version 2 vault behavior.**
+
+- **Debt Allocator:** Knows the maximum debt a vault can send to a strategy.
+- **Strategy Registry:** Handles adding and removing strategies of a vault. It can also include strategy metadata.
+- **Fee Manager:** Handles changing fees for vault operations.
+- **HealthCheck:** Guardrail vault operations so that profit & loss reporting is always under acceptable values.
+
+## Peripheral Modules
+
+Peripheral Modules are a separate layer of optional functions for the vault and strategies. They are not required but they facilitate building around yVaults and creating new strategies.
+
+- **yRouter**: Wrapper that handles deposits and withdrawals to/from the vault.
+- **Yearn Lens:** Information aggregator for off-chain apps.
+- **APY TWAP Oracle:** Reliable source of Yearn vaults’ past performance.
+- **ySwaps:** Internal swap system currently being developed. Reduces slippage thus improving net APY.
+- And any others you can come up with!
+
+# Version 3 yVault Specification
+
+## Definitions
+- **Asset:** Any ERC20-compliant token
+- **Shares:** ERC20-compliant token that tracks Asset balance in the vault for every distributor. Named yv<Asset_Symbol>
+- **Depositor:** Account that holds Shares
+- **Strategy:** Smart contract used to deposit in Protocols to generate yield
+- **Vault:** ERC4626 compliant smart contract that receives Assets from Depositors to distribute amongst the different Strategies added to the vault: manages accounting and Assets distribution. 
```

Comment:
`manages accounting` you say manages accounting but there is an Accountant smart contract?

Reply:
I think @jmonteer meant that the vault handles accounting logic for all users

Reply:
https://github.com/yearn/yearn-devdocs/pull/291/files#diff-7fc26d081f78daefe7b3cf46c856f64bb949779358599412cd3b2250ab13bab8R38